pyproject.toml

```diff
@@ -39,13 +39,17 @@ classifiers = [
 dynamic = [
   "version",
 ]
+dependencies = [
```

Comment:
I, personally, would just make it `23.1.2` everywhere...

Reply:
We don't actively test with other dependencies in the CI, so would soon be inaccurate anyway.

Reply:
Sure! I specified the most correct dependencies just in case, but happy to just put `pip>=23.1.2` :slightly_smiling_face:

Reply:
Done.

Reply:
Just a small side note; we do handle older versions of pip in the code (from what I can see it's with the [discovery](https://github.com/tox-dev/pipdeptree/blob/bd4453bc02a10c22a3d5c5e6a18ef86075909278/src/pipdeptree/_discovery.py#L16C1-L19C90)-related and  [frozen requirements](https://github.com/tox-dev/pipdeptree/blob/bd4453bc02a10c22a3d5c5e6a18ef86075909278/src/pipdeptree/_models/package.py#L77C1-L89C121)-related code). If this won't cause any problems, we can probably update these areas to just use the API that is supported in 23.1.2 (or wait for https://github.com/tox-dev/pipdeptree/pull/333 to get merged).